### PR TITLE
8.13 update top

### PR DESCRIPTION
**Pull request by jdu2600**
Update top documentation for 8.13.0

### CODE REVIEW

**Review by AsuNa-jp — APPROVED** [review]

File: EndpointTopCommand.md [hunk 1]
```diff
@@ -17,28 +18,28 @@ In other words, if Endpoint is consuming CPU, it's likely in response to some ot
...
+| vctip.exe          |     0.0 |  0.0 |  0.0 |       0.0 | 0.0 |  0.0 | 0.0 |  0.0 |      0.0 | 0.0 |  0.0 |    0.0 | 0.0 | 0.0 |
+| mscorsvw.exe       |     0.0 |  0.0 |  0.0 |       0.0 | 0.0 |  0.0 | 0.0 |  0.0 |      0.0 | 0.0 |  0.0 |    0.0 | 0.0 | 0.0 |
+| msedgewebview2.exe |     0.0 |  0.0 |  0.0 |       0.0 | 0.0 |  0.0 | 0.0 |  0.0 |      0.0 | 0.0 |  0.0 |    0.0 | 0.0 | 0.0 |
+| filebeat.exe       |     0.0 |  0.0 |  0.0 |       0.0 | 0.0 |  0.0 | 0.0 |  0.0 |      0.0 | 0.0 |  0.0 |    0.0 | 0.0 | 0.0 |
+| devenv.exe         |     0.0 |  0.0 |  0.0 |       0.0 | 0.0 |  0.0 | 0.0 |  0.0 |      0.0 | 0.0 |  0.0 |    0.0 | 0.0 | 0.0 |
+| Tracker.exe        |     0.0 |  0.0 |  0.0 |       0.0 | 0.0 |  0.0 | 0.0 |  0.0 |      0.0 | 0.0 |  0.0 |    0.0 | 0.0 | 0.0 |
+| link.exe           |     0.0 |  0.0 |  0.0 |       0.0 | 0.0 |  0.0 | 0.0 |  0.0 |      0.0 | 0.0 |  0.0 |    0.0 | 0.0 | 0.0 |
 
 Endpoint service (16 CPU): 44.3% out of 1600%
 ```
```
[the remark this answers is not in the document]
Comment (jdu2600) [reply EndpointTopCommand.md:45]: The screenshot is from `8.12` - and we don't intent to update it each version.

**Review by nfritts — APPROVED** [review]

File: EndpointTopCommand.md [hunk 2]
```diff
@@ -92,22 +93,21 @@ To fit everything on the screen, columns are abbreviated as follows:
...
+| MEM SCAN | Memory Scanning (part of Memory Protection) | Uncheck [Memory threat protections enabled](https://www.elastic.co/guide/en/security/current/configure-endpoint-integration-policy.html#memory-protection) to turn off Memory Protection entirely, or set `*.advanced.memory_protection.memory_scan: false` in [Advanced Policy](https://www.elastic.co/guide/en/security/current/configure-endpoint-integration-policy.html#adv-policy-settings) to turn off only Memory Scanning. |
+| BHVR | Malicious Behavior Protection (Rules Engine) | Uncheck [Malicious behavior protections enabled](https://www.elastic.co/guide/en/security/current/configure-endpoint-integration-policy.html#behavior-protection) |
+| DIAG BHVR | Diagnostic Malicious Behavior Protection (Rules Engine) | Set `*.advanced.diagnostic.enabled: false` in [Advanced Policy](https://www.elastic.co/guide/en/security/current/configure-endpoint-integration-policy.html#adv-policy-settings) |
```
Comment (intxgo) [inline EndpointTopCommand.md:110]: nice, I'm not sure how the "current" links will behave if we deprecate (remove) something in the future. We should remember to update this doc with such change.

**Review by intxgo — APPROVED** [review]

File: EndpointTopCommand.md (hunk 2 above)
Comment (gabriellandau) [reply EndpointTopCommand.md:110]: That's a good point.  We should probably version pin these.

Comment (jdu2600) [reply EndpointTopCommand.md]: 👍 30faedd

**Review by gabriellandau — APPROVED** [review]

**Pull request merged**